### PR TITLE
Project 5 - Ati Takrami 

### PR DESCRIPTION
**Pull request by takrami**
I covered all the levels.
- [x] Blue
- [x] Red
- [x] Black

### CODE REVIEW

**Review by vladjnbykov — APPROVED** [review]
Great job, Ati! Nice to play with many interesting features. Not really so much to add, but anyway:
Design-wise it would be great to add some extra effect at the end of the game depending whether person is winning or losing, like sound or some dramatic changes in the layout, in order to enhance user-experience.
README file: I would rather remove original statements and replace them with some answers, like which tech was used etc.
General comment on coding: practically where are no comments in the code. It would be good to use them periodically, so that the new person will quickly become accustomed to the code. It is even good to do it for own sake because if one returns after let say 1 year to the same project it might take a long time to figure out what each function does.
Nevertheless, these are very minor points. Great Job!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
+# Guess who
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://affectionate-colden-2160ca.netlify.app/
```
Comment (KarolCislowski) [inline README.md:7]: Your project shines like a diamond 💎  but nobody will know about it if you will not have a great invitation here. ReadMe file is the cover of your code. It is the first impression. Polish it as hard as you polished your code 🎨

File: code/assets/css/animations.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+@keyframes loader {
```
Comment (KarolCislowski) [inline code/assets/css/animations.css:1]: great animation!

File: code/assets/css/intro.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
+@import "./normalize.css";
+@import "./variables.css";
+@import "./general.css";
+@import "./animations.css";
+
```
Comment (KarolCislowski) [inline code/assets/css/intro.css:5]: https://greensock.com/gsap/ I think you can like it a lot 😸

**Review by KarolCislowski — APPROVED** [review]
This is a very impressive project! 🌈  You have shown great ambition 🦄 

Work on your ReadMe and there will be nothing to squeeze to from this project 🥳

**Pull request closed without merging**